api/v1alpha1/drpolicy_types.go

```diff
@@ -20,13 +20,46 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// DR cluster information
+type DRClusterInfo struct {
+	// Name of this ManagedCluster as configured in OCM/ACM
+	ManagedClusterName string `json:"managedClusterName"`
+
+	// Specifies the name of the S3 profile (in RamenConfig) that is expected to
```

Comment:
We can reword this field definition as follows:

```
S3 profile name (in Ramen config) to use as a source for PV metadata restore during failover or relocate actions. The S3 profile mentioned is expected to be available when the workload is moved to the ManagedClusterName.
NOTE: PV metadata from this ManagedClusterName, when the applications are active on it, is stored to all other S3 profiles mentioned in other ManagedClusterName's.
```

Reply:
Thanks, will do.